etl/steps/data/garden/worldbank_wdi/2024-05-20/wdi.py

```diff
@@ -110,7 +102,7 @@ def mk_omms(table: Table) -> Table:
         title="Urban population living in slums",
         description=(
             "Total urban population living in slums. This variable is calculated"
-            f" by Our World in Data based on the following variables from {source.published_by}:"
+            f" by Our World in Data based on the following variables from {published_by}:"
             f' "{table[urb_pop_code].metadata.title}"'
             " and"
             f' "{table[slums_pct_code].metadata.title}"'
```

Comment:
Sometimes (e.g. omm_urb_pop_slum) the added description is "None". It would be better if this text is only added if the description is not None (but I'm no sure at this point if this can affect other things down the line).